tuned/plugins/plugin_irq.py

```diff
@@ -0,0 +1,167 @@
+from . import hotplug
+from .decorators import *
+import tuned.consts as consts
+import tuned.logs
+from tuned.utils.commands import commands
+
+import errno
+import os
+
+log = tuned.logs.get()
+
+class IrqPlugin(hotplug.Plugin):
+
+	def __init__(self, monitor_repository, storage_factory, hardware_inventory, device_matcher, device_matcher_udev, plugin_instance_factory, global_cfg, variables):
+		super(IrqPlugin, self).__init__(monitor_repository, storage_factory, hardware_inventory, device_matcher, device_matcher_udev, plugin_instance_factory, global_cfg, variables)
+		self._irq_original = {}
+		self._irq_unchangeable = []
+
+	def _init_devices(self):
+		self._devices_supported = True
+		self._free_devices = set()
+		self._assigned_devices = set()
```

Comment:
Could you match using udev? Like other device plugins do. I think it could work (I didn't test it), because e.g.:
```
# udevadm info /sys/kernel/irq/59
P: /kernel/irq/59
M: 59
```